screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -218,21 +235,8 @@ async fn main() -> anyhow::Result<()> {
         _ => true,
     };
 
-    // Override logging state if debug console is enabled
-    #[cfg(feature = "debug-console")]
-    let should_log = if cfg!(all(debug_assertions, feature = "debug-console")) {
-        console_subscriber::init();
-        false
-    } else {
-        should_log
-    };
-
     // Store the guard in a variable that lives for the entire main function
-    let _log_guard = if should_log {
```

Comment:
`screenpipe pipe list -o json | jq` still avoid showing logs?

Reply:
this needs to be fixed